src/Microsoft.Health.Fhir.Core/Features/Operations/Import/GetBulkImportRequestHandler.cs

```diff
@@ -0,0 +1,84 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Core.Features.Security.Authorization;
+using Microsoft.Health.Fhir.Core.Exceptions;
+using Microsoft.Health.Fhir.Core.Features.Security;
+using Microsoft.Health.Fhir.Core.Messages.Import;
+using Microsoft.Health.TaskManagement;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Import
+{
+    public class GetBulkImportRequestHandler : IRequestHandler<GetImportRequest, GetImportResponse>
```

Comment:
Can this be renamed to GetImportRequestHandler to fit with the pattern that the Handler, Request, and Response all have the same prefix?

Reply:
Updated. Thanks.